src/components/Header/Header.module.scss

```diff
@@ -1,20 +1,182 @@
+@import '../../variables.scss';
+
 .list {
   display: flex;
   list-style: none;
 }
-
 .link {
   margin-left: 1rem;
-
   & a {
     color: grey;
     text-decoration: none;
   }
-
   & a:hover,
   a:active,
   a.active {
     color: black;
-    border-bottom: 4px solid grey;
+    border-bottom: 4px solid $green;
+  }
+}
+///////////////////////////////////////
+
+.header {
+  @include flex-column;
+}
+
+.info {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 3rem;
+  height: 3rem;
+
+  & a {
+    @include open-sans;
+    font-weight: 400;
+    font-size: 0.75rem;
+    line-height: 1rem;
+    text-decoration: none;
+    cursor: pointer;
+  }
+
+  &__left,
+  &__right {
+    @include flex-centerY;
+  }
+
+  &__left a {
+    margin-right: 2.37rem;
+  }
+  &__right a {
+    margin-left: 2.37rem;
+  }
+
+  &__link {
+    @include link;
+  }
+
+  &__info {
+    color: black;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
+
+.line {
+  border-top: 1px solid #eeeeee;
+  border-bottom: none;
+  margin: 0 3rem;
+}
+
+.main {
+  height: 8.81rem;
+  padding: 0.81rem 1.68rem 0.81rem 3.87rem;
+  @include flex-column;
+  justify-content: space-between;
+  background: $lighter-grey;
+
+  &__top {
+    @include flex-centerY;
+    justify-content: space-between;
+    height: 50%;
+  }
+
+  &__logo {
+    @include flex-centerY;
+  }
+
+  &__search {
+    height: 2.62rem;
+    width: 50%;
+    max-width: 30rem;
+    border-radius: 12px;
+    @include flex-centerY;
+    justify-content: space-evenly;
+    border: 1px solid $lines-grey;
+    background: $bg-grey;
+    padding: 0.5rem 1rem;
+  }
+
+  &__categories {
+    @include flex-centerY;
+    cursor: pointer;
+    white-space: nowrap;
+  }
+  &__categories span {
+    @include poppins;
+    font-weight: 700;
+    margin-right: 0.7rem;
+    font-size: 0.95rem;
+  }
+  &__categories svg {
+    color: $green;
+    font-size: 0.6rem;
+    margin-right: 1.2rem;
+  }
+
+  &__divider {
+    border-left: 1px solid $lines-grey;
+    height: 1.3rem;
+  }
+
+  &__input {
+    margin-left: 1.2rem;
+    @include flex-centerY;
+    justify-content: space-between;
+    position: relative;
+    width: 60%;
+  }
+  &__input input {

```

Comment:
Bad naming, because you duplicate html tag, it could be container or smth like that